cmd/root.go

```diff
@@ -161,6 +161,20 @@ func init() {
 		"Set a request timeout for the client to connect with Kong (in seconds).")
 	viper.BindPFlag("timeout",
 		rootCmd.PersistentFlags().Lookup("timeout"))
+
+	rootCmd.PersistentFlags().String("tls-client-cert", "",
+		"Set the Kong's Admin TLS client certificate.\n"+
+			"This value can also be set using DECK_TLS_CLIENT_CERT "+
+			"environment variable. Must be used in conjunction with tls-client-key")
+	viper.BindPFlag("tls-client-cert",
+		rootCmd.PersistentFlags().Lookup("tls-client-cert"))
+
+	rootCmd.PersistentFlags().String("tls-client-key", "",
```

Comment:
Same here, can you please append `-file`?

Reply:
The expected content is a string representing the certificate, so would you rather think about passing a file instead ?
I will renamed then.

Reply:
Done.